### DISCUSSION TITLE
docs <class 'AttributeError'> NetBox 3.7.4

### DISCUSSION THREAD

**Discussion by gitwpia** [netbox-community/netbox#15472] [Help Wanted!]
Hello
I have prolem with Docs (netbox work OK)
When I click Docs on Dashboard - myIP:8000/static/docs/ I see the error:
```
Server Error
<class 'AttributeError'>
'WSGIRequest' object has no attribute 'user'
Python version: 3.9.2
NetBox version: 3.7.4
Plugins: None installed
```
When I add index.html (myip:8000/static/docs/index.html) it's work OK
but when I click something from left menu there is the same error and I must add index.html (myIP:8000/static/docs/installation/index.html)

How to repair it?
Thank you

**Comment by candlerb** [comment]
Are you talking to the development server directly, instead of running a front-end proxy like Apache or Nginx? If so, that's almost certainly the problem: the development server doesn't have good support for static files (such as mapping a directory to index.html).

Running with `DEBUG=True` in configuration.py confirms this:

![image](https://github.com/netbox-community/netbox/assets/44789/bb568f35-9b55-4c79-8e7c-595a4670e28d)

(I don't know where `'WSGIRequest' object has no attribute 'user'` comes from, probably somewhere in the request handling chain).

In any case, the solution is to [deploy Netbox for production](https://docs.netbox.dev/en/stable/installation/5-http-server/). You can see how it works properly on the demo site: https://demo.netbox.dev/static/docs/

**Reply by gitwpia** [reply]
I have the same page.
I don't have apache or nginx.

I have to live with it :)
Thank you
